app/components/Dashboard/PriceHistoryPanel/PriceHistoryPanel.jsx

```diff
@@ -39,11 +44,17 @@ type Props = {
   priceKey: string
 }
 
-const DURATIONS: { [key: Duration]: string } = {
-  '1m': '1 month',
-  '1w': '1 week',
-  '1d': '1 day'
-}
+const DURATIONS: Array<[Duration, string]> = [
+  ['1d', '1 DAY'],
+  ['1w', '1 WEEK'],
+  ['1m', '1 MONTH']
+]
```

Comment:
Went for Tuple array instead of Object to guarantee entry order.

Didn't go for `new Map()` since it's the same syntax (gets exact array as argument) and iteration is more cumbersome.

Reply:
@ranbena Just curious why a Tuple array was chosen instead of a collection as this also requires the map you've added below.

Reply:
Hey @zsiglin, here's my thought process:

### Option 1 - No mapping
```
DURATION_OPTIONS: Array<SelectOption> = [
  { value: '1d', label: '1 DAY' },
  { value: '1w', label: '1 WEEK' },
  { value: '1m', label: '1 MONTH' }
]
```
### Option 2 - Array
```
const DURATIONS: Array<[Duration, string]> = [
  ['1d', '1 DAY'],
  ['1w', '1 WEEK'],
  ['1m', '1 MONTH']
]

const DURATION_OPTIONS: Array<SelectOption> = DURATIONS.map(([k, v]) => ({
  value: k,
  label: v
}))
```
### Option 3 - Map
```
const DURATIONS: Map<Duration, string> = new Map([
  ['1d', '1 DAY'],
  ['1w', '1 WEEK'],
  ['1m', '1 MONTH']
])

const DURATION_OPTIONS: Array<SelectOption> = Array.from(DURATIONS, ([ k, v ]) => ({
  value: k,
  label: v
}))
```

### Considerations
Option 1 is best for perf, simpler and has less code. Only downside is that it doesn't adhere to "separation of data and implementation" school of thought. Not optimal if in future dropdown component is swapped for another.

Option 2 is not as readable as the original Object `{'1d': '1 DAY', ...}` but does separate data/impl.

Option 3 has almost identical syntax as the same as previous, mildly lengthier. I see no advantage to it.